presets/mainnet/electra.yaml

```diff
@@ -43,3 +43,8 @@ MAX_WITHDRAWAL_REQUESTS_PER_PAYLOAD: 16
 # ---------------------------------------------------------------
 # 2**3 ( = 8) pending withdrawals
 MAX_PENDING_PARTIALS_PER_WITHDRAWALS_SWEEP: 8
+
+# Pending deposits processing
+# ---------------------------------------------------------------
+# 2**4 ( = 4) pending deposits
+MAX_PENDING_DEPOSITS_PER_EPOCH_PROCESSING: 16
```

Comment:
I understand this limit is specifically for epoch processing. Does it need to emphasize the epoch processing part? Asking because there's no precedent on presets having `_PER_EPOCH_PROCESSING` suffix but `_PER_EPOCH`